src/flow/io/event.js

```diff
@@ -310,16 +310,20 @@ export function importCalendarEvents(calendar, user, defaultEvents) {
   const { type, data, name } = calendar || {}
   let fn = () => {}
   let config
+  let fetchParam = data.src
   if (type === 'ics') {
     fn = fetchAndParseIcal
+    fetchParam = data.events
+  } else if (type === 'ics-raw') {
+    fn = parseIcal
   } else if (type === 'blockstack-user') {
     config = { decrypt: false, username: data.user }
     fn = fetchFromBlockstack
   } else if (type === 'private') {
     fn = fetchFromBlockstack
   }
 
-  return fn(data.src, config)
+  return fn(fetchParam, config)
```

Comment:
probably `fetchParam` can go into config

Reply:
Why?

Reply:
fetchParam is the source... it's not a configuration..

Reply:
fn is so generic that fetchParam does not make sense, e.g. is not used by fetchFromBlockstack

Reply:
I moved the events to the config param... Is better now?